src/test/utils.js

```diff
@@ -46,7 +46,10 @@ export const getContext = state => {
   return { store };
 };
 
-export const store = configureStore(Immutable.Map(), history);
+// eslint-disable-next-line no-shadow
+export const createStore = ({ initialState = Immutable.Map(), history = history }) => configureStore(initialState, history);
```

Comment:
Is this export used anywhere? Doesn't seem so.

Reply:
Not currently. But it could be :) When you need to supply your own store with different initial state or custom history.